src/content/toolbars.html

```diff
@@ -1043,30 +1043,156 @@
+					<input type="checkbox" />
+				</div>
+				<div class="toolbar-group-content">
+					<div class="dropdown">
+						<a class="dropdown-toggle" data-toggle="dropdown" href="#1">
+					Order by: <span class="icon-monospaced icon-sort"></span>
+						</a>
+						<ul class="dropdown-menu">
+							<li><a href="#1">Date</a></li>
+							<li><a href="#1">Time</a></li>
+							<li><a href="#1">Status</a></li>
+						</ul>
+					</div>
+				</div>
+				<div class="toolbar-group-content">
+					<div class="dropdown">
+						<a class="dropdown-toggle" data-toggle="dropdown" href="#1">Name: <span class="icon-monospaced icon-sort"></span></a>
+						<ul class="dropdown-menu dropdown-menu-right">
+							<li><a href="#1">Alphabetical</a></li>
+							<li><a href="#1">Reverse</a></li>
+							<li><a href="#1">Split</a></li>
+						</ul>
+					</div>
+				</div>
+				<div class="toolbar-group-content">
+					<a class="btn btn-default" href="#1">
+						<span class="icon-monospaced icon-caret-up" />
+					</a>
+				</div>
+				<div class="toolbar-group-content">
+					<a class="btn btn-default" href="#1">
+						<span class="icon-monospaced icon-caret-down" />
+					</a>
+				</div>
+			</div>
+			<div class="toolbar-group-right">
+				<div class="toolbar-group-content">
+					<a class="active btn btn-default" href="#1">
+						<span class="icon-monospaced icon-th-large" />
+					</a>
+				</div>
+				<div class="toolbar-group-content">
+					<a class="btn btn-default" href="#1">
+						<span class="icon-monospaced icon-th-list" />
+					</a>
+				</div>
+				<div class="toolbar-group-content">
+					<a class="btn btn-default" href="#1">
+						<span class="icon-monospaced icon-th" />
+					</a>
+				</div>
+			</div>
+		</div>
+
+<div class="uxgl-toggle-code">
+	<a class="btn btn-warning btn-xs" data-toggle="collapse" href="#codeCollapse11"><span class="icon-caret-down"></span></a>
+</div>
+<div class="collapse" id="codeCollapse11">
+<pre><code>```<div class="management-toolbar-default toolbar">
+			<div class="toolbar-group">
+				<div class="toolbar-group-content">
+					<input type="checkbox" />
+				</div>
+				<div class="toolbar-group-content">
+					<div class="dropdown">
+						<a class="dropdown-toggle" data-toggle="dropdown" href="#1">
+					Order by: <span class="icon-monospaced icon-sort"></span>
+						</a>
+						<ul class="dropdown-menu">
+							<li><a href="#1">Date</a></li>
+							<li><a href="#1">Time</a></li>
+							<li><a href="#1">Status</a></li>
+						</ul>
+					</div>
+				</div>
+				<div class="toolbar-group-content">
+					<div class="dropdown">
+						<a class="dropdown-toggle" data-toggle="dropdown" href="#1">Name: <span class="icon-monospaced icon-sort"></span></a>
+						<ul class="dropdown-menu dropdown-menu-right">
+							<li><a href="#1">Alphabetical</a></li>
+							<li><a href="#1">Reverse</a></li>
+							<li><a href="#1">Split</a></li>
+						</ul>
+					</div>
+				</div>
+				<div class="toolbar-group-content">
```

Comment:
This and the next toolbar-group-content should be placed inside a separate toolbar.